src/-private/tracking.ts

```diff
@@ -1,48 +1,50 @@
-import { tracked } from '@glimmer/tracking';
-
-class Tag {
-  @tracked private __tagValue__: undefined;
-
-  static consumeTag(tag: Tag): void {
-    // read the tag value
-    tag.__tagValue__;
-  }
-
-  static dirtyTag(tag: Tag): void {
-    // write the tag value
-    tag.__tagValue__ = undefined;
-  }
+import {
+  createStorage,
+  getValue,
+  setValue,
+  TrackedStorage,
+} from 'ember-tracked-storage-polyfill';
```

Comment:
I guess this implies that using this library now only works in Ember, right? (it uses for example `@ember/debug`).

@pzuraq is compatibility of this lib with GlimmerJS still a concern? AFAICT there are no tests for that scenario anyway. Or can we make it a "real" Ember addon?